test/functional/wallet_listsinceblock.py

```diff
@@ -448,6 +449,19 @@ def test_send_to_self(self):
         assert any(c["address"] == addr for c in coins)
         assert all(self.nodes[2].getaddressinfo(c["address"])["ischange"] for c in coins)
 
+    def test_label(self):
+        self.log.info("Test label")
+        new_addr = self.nodes[1].getnewaddress("new_addr", "bech32")
+
+        self.nodes[2].sendtoaddress(new_addr, "0.001")
+        block_hash = self.generate(self.nodes[2], 1)[0]
+
+        new_addr_transactions = self.nodes[1].listsinceblock(label="new_addr", blockhash=block_hash)["transactions"]
+        for transaction in new_addr_transactions:
+            assert_equal(transaction["label"], "new_addr")
+
+        assert_equal(self.nodes[1].listsinceblock(label="", blockhash=block_hash), self.nodes[1].listsinceblock(blockhash=block_hash))
```

Comment:
I think the is supposed to be like this:
```suggestion
        assert_equal(self.nodes[1].listsinceblock(label=None, blockhash=block_hash), self.nodes[1].listsinceblock(blockhash=block_hash)) # probably not needed because it is redundant

        empty_label_transactions = self.nodes[1].listsinceblock(label="", blockhash=block_hash)["transactions"]
        for transaction in empty_label_transactions:
            assert_equal(transaction["label"], "")
```